dt-assets/functions/details-bar.php

```diff
@@ -112,8 +118,8 @@ class="second-bar" id="second-bar-large">
                         <?php if ( $share_button ): ?>
                         <div class="cell shrink center-items ">
                             <button class="center-items open-share">
-                                <img class="dt-blue-icon" src="<?php echo esc_url( get_template_directory_uri() . "/dt-assets/images/share.svg" ) ?>">
-                                <span style="margin:0 10px 2px 10px"><?php esc_html_e( "Share", "disciple_tools" ); ?></span>
+                                <img class="dt-blue-icon" src="<?php echo esc_url( get_template_directory_uri() . "/dt-assets/images/share.svg" ); ?>">
+                                <span data-tooltip title="<?php echo esc_html( sprintf( '%s', ltrim( $shared_with_text, ',' ) ) ); ?>" style="margin:0 10px 2px 10px"><?php echo sprintf( 'Share (%s)', esc_html( count( $shared_with ) ) ); ?></span>
```

Comment:
Hey @cwuensche, looks like "share" might no longer be translated

I'd suggest
`<?php echo sprintf( _x( 'Share (%s)', 'Share (23)', 'disciple_tools' )  , esc_html( count( $shared_with ) ) ); ?>`
or 
`<?php esc_html_e( "Share", "disciple_tools" ); ?> (<?php echo esc_html( count($shared_with) ); ?>)`
I think i might prefer the second one just because it is already translated and it is easier to translate.

Reply:
> Hey @cwuensche, looks like "share" might no longer be translated
> 
> I'd suggest
> `<?php echo sprintf( _x( 'Share (%s)', 'Share (23)', 'disciple_tools' ) , esc_html( count( $shared_with ) ) ); ?>`
> or
> `<?php esc_html_e( "Share", "disciple_tools" ); ?> (<?php echo esc_html( count($shared_with) ); ?>)`
> I think i might prefer the second one just because it is already translated and it is easier to translate.

Went ahead with the second solution.